Dockerfile

```diff
@@ -79,10 +90,32 @@ RUN git clone --depth 1 https://github.com/mongodb/devhub.git snooty-devhub
     && cd snooty-devhub                                                                            \
     && npm install --production
 
+# install redoc fork
+RUN git clone -b ${REDOC_CLI_VERSION} --depth 1 https://github.com/mongodb-forks/redoc.git redoc \
+    # Install dependencies for Redoc CLI
+    && cd redoc/cli \
+    && npm ci --omit=dev \
+    # Install dependencies for Redoc component and building/compiling
+    && cd ../ \
+    && npm ci --omit=dev --ignore-scripts \
+    && npm run compile:cli
+
 COPY --from=ts-compiler /home/docsworker-xlarge/package*.json ./
 COPY --from=ts-compiler /home/docsworker-xlarge/config config/
 COPY --from=ts-compiler /home/docsworker-xlarge/build ./
 RUN npm install
+
+# OAS Page Builder module copy
+# Create directory and add permissions to allow node module installation
+RUN mkdir -p modules/oas-page-builder && chmod 755 modules/oas-page-builder
+COPY --from=ts-compiler /home/docsworker-xlarge/modules/oas-page-builder/package*.json ./modules/oas-page-builder/
+COPY --from=ts-compiler /home/docsworker-xlarge/modules/oas-page-builder/dist ./modules/oas-page-builder/
+RUN cd ./modules/oas-page-builder/ && npm install
```

Comment:
is the installation necessary again? see that we are installing in line 13-15?

also see from [OAS readme](https://github.com/mongodb/docs-worker-pool/blob/master/modules/oas-page-builder/README.md#running) that we should be using npm commands

Reply:
oh i see, previously installing and building in original directory, then transferring from repo dir to home dir? do we have to install the first time around then?

Reply:
> also see from [OAS readme](https://github.com/mongodb/docs-worker-pool/blob/master/modules/oas-page-builder/README.md#running) that we should be using npm commands

I'm a little confused by which npm commands you're referring to. Running with `npm start` isn't really applicable in this case with building.

> previously installing and building in original directory, then transferring from repo dir to home dir?

More or less. We're building the module in the `ts-compiler` stage, and then copying only the needed files over from the modules directory of that stage to the modules directory our current/final stage.

> do we have to install the first time around then?

I'm actually unsure about this. The first installation ensures that anything needed to successfully build the module is installed. Testing locally, it seems like it might be okay to leave out the first installation since we don't really have anything special about the build step, and `tsc` should already be installed. Will push up a new commit later

Reply:
thanks @rayangler !